libraries/node-core-library/src/Path.ts

```diff
@@ -98,6 +148,60 @@ export class Path {
     return absolutePath;
   }
 
+  /**
+   * Formats a file location to look nice for reporting purposes.
+   * @remarks
+   * If `pathToFormat` is under the `baseFolder`, then it will be converted to a relative with the `./` prefix.
+   * Otherwise, it will be converted to an absolute path.
+   *
+   * Backslashes will be converted to slashes, unless the path starts with an OS-specific string like `C:\`.
+   */
+  public static formatFileLocation(options: IPathFormatFileLocationOptions): string {
+    const { message, format, line, column } = options;
```

Comment:
```suggestion
    const { message, format, line, column, baseFolder, pathToFormat } = options;
```

And then use these below.